test/cases/koji.sh

```diff
@@ -46,10 +46,10 @@ sudo cp \
     "${OSBUILD_COMPOSER_TEST_DATA}"/kerberos/krb5-local.conf \
     /etc/krb5.conf.d/local
 
-greenprint "Adding generated CA cert for Koji"
+greenprint "Adding the testsuite's CA cert to the system trust store"
 sudo cp \
-    /tmp/osbuild-composer-koji-test/ca-crt.pem \
-    /etc/pki/ca-trust/source/anchors/koji-ca-crt.pem
+    /etc/osbuild-composer/ca-crt.pem \
+    /etc/pki/ca-trust/source/anchors/osbuild-composer-tests-ca-crt.pem
```

Comment:
I remember doing this out of laziness, is there a way to drop in the right config to point to /etc/osbuild-composer/ca-crt.pem directly instead?

Reply:
`koji` can be indeed run with a custom CA but I didn't want to make more changes in this PR. Will do in a follow-up!

Reply:
Thanks!